README.md

```diff
@@ -2,7 +2,7 @@
 # Flow Visualization
 
 This is a [React] application to generate flow graphs from [JSON](https://www.json.org/) or [YAML](https://yaml.org/).
-It is built on top of [ReactFlow].
+It is built on top of [ReactFlowComponent].
```

Comment:
This is probably a faulty refactoring result ;)

Reply:
Your vision is insane :-).

Fixed in https://github.com/KNowledgeOnWebScale/dataflow-visualization/commit/ee04060c3f3831d746a8d27dcb096d5ef429f470